extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfigPersistenceUnit.java

```diff
@@ -107,6 +107,12 @@ public boolean isAnyPropertySet() {
         @ConfigItem(name = ConfigItem.PARENT, defaultValue = "none")
         public String generation = "none";
 
+        /**
+         * Specifies the statement delimiter to be used for the generated DDL scripts.
+         */
+        @ConfigItem(defaultValue = ";")
```

Comment:
Does it have a new line by default? If not, I would make it `;\n` by default so that the scripts are more readable.

That being said, I think ORM itself should have sensible defaults depending on the dialect. That would make more sense than having to define this at the Quarkus level.

@Sanne can you point me to the person who implemented this so that we can start a discussion about it and make progress before 2.0? If we can't, I suppose this patch can be merged anyway.

Reply:
> Does it have a new line by default? If not, I would make it `;\n` by default so that the scripts are more readable.

Yes, Hibernate itself already adds line terminators after the statement delimiters.

> That being said, I think ORM itself should have sensible defaults depending on the dialect. That would make more sense than having to define this at the Quarkus level.

I would also be curious as to why Hibernate itself doesn't add statement delimiters to the generated scripts unless explicitly configured.

Reply:
Current status of this one: we should have `;` as the default in Hibernate ORM for 2.0.0.Final.

Wondering if it's worth adding the config property?

Reply:
I would prefer to not have the property - even if there was in future some obscure database not supporting the current default, I'd rather have Quarkus switch defaults automatically based on dialect/vendor.

Reply:
ah apparently I had already suggested it:
 - https://github.com/quarkusio/quarkus/issues/17514#issuecomment-850285107

Good, I will feel less nasty in closing your PR @knutwannheden :)

But thank you so much anyway, it's great to see more people helping!